RentACar/RentACar/Helpers/CrudHelper.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+
+namespace RentACar.Utilities
+{
+    static public class CrudHelper
+    {
+        public static Car UpdateCarInListById(List<Car> cars, Car car)
+        {
+            var carUpdated = cars.FirstOrDefault(e => e.Id == car.Id);
+
+            carUpdated.Brand = car.Brand;
+            carUpdated.Color = car.Color;
+            carUpdated.DoorsQuantity = car.DoorsQuantity;
+            carUpdated.Model = car.Model;
+            carUpdated.Transmission = car.Transmission;
+
+            return carUpdated;
+        }
```

Comment:
this looks like something that belongs to the CRUD itself and not a helper, what do you think?

Reply:
Definitely yes! i was excited with my new helper "class" and i put all inside them 😛..
If in the future we will have other types like "Car", should i create another class to group thoose methods?
By "thoose methods" i mean the methods that match each property.

Reply:
let's wait and see how this evolve, don't try to solve a problem you still don't have (but it's always good to keep in mind things like the ones you mentioned)

Reply:
The famous over-engineering! Okey, thanks!